crates/ibc/src/applications/transfer/amount.rs

```diff
@@ -15,7 +15,7 @@ use crate::alloc::{borrow::ToOwned, string::String};
 #[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Display, From, Into)]
 pub struct Amount(
     #[cfg_attr(feature = "schema", schemars(with = "String"))]
-    #[serde(serialize_with = "crate::serializers::serde_string::serialize")]
+    #[serde(serialize_with = "serialize")]
```

Comment:
Taking another peek, the "serialize" function in "amount.rs" seems redundant. Why not use "crate::serializers::serde_string::serialize"?

Reply:
afaik for some reason it triggered `float` to be compiled into wasm (seems like just patter matching, but validation tools do not care).

Reply:
Would be OK if the `ci/cw-check` passes with the `crate::serializers::serde_string::serialize` or still might be the case?

Reply:
would be OK

Reply:
i used only reproducible check to fix issues